web/lib/roadmap/roadmapBuilder.ts

```diff
@@ -25,6 +26,29 @@ export const buildRoadmap = async ({
   return convertToRoadmap(roadmapBuilder);
 };
 
+const generateIndustryRoadmap = async (
+  builder: RoadmapBuilder,
+  industryId: string,
+  addOns: string[],
+  modifications: string[]
+): Promise<RoadmapBuilder> => {
+  let industryRoadmap: IndustryRoadmap;
+  if (process.env.NODE_ENV === "test") {
+    industryRoadmap = (await import(`@/lib/roadmap/fixtures/industries/${industryId}.json`))
+      .default as IndustryRoadmap;
+  } else {
+    industryRoadmap = (await import(`../../roadmaps/industries/${industryId}.json`))
+      .default as IndustryRoadmap;
+  }
```

Comment:
Can we break this into a helper function like the other similar ones? The if/else necessity for testing really looks ugly which is why I hid them all in functions at the bottom of the file to make the logical flow as readable as possible

Reply:
Refactored